### PR TITLE
Add Act as a Morse Code Translator

### PR DESCRIPTION
**Pull request by iuzn**
# Add Act as a Morse Code Translator

Example: https://sharegpt.com/c/yugHfth

- [x] I've confirmed the prompt works well
- [x] I've added `Contributed by: [@yourusername](https://github.com/yourusername)`
- [x] I've added to the README.md
- [x] I've added to the `prompts.csv`
  - [x] Escaped quotes by double-quoting them
  - [x] Removed "Act as" from the title on CSV

### CODE REVIEW

**Comment by iuzn** [comment]
@f, I fixed the bug that existed in the previous Pull Request.

File: prompts.csv [hunk 1]
```diff
@@ -125,4 +125,4 @@
...
 "Tic-Tac-Toe Game","I want you to act as a Tic-Tac-Toe game. I will make the moves and you will update the game board to reflect my moves and determine if there is a winner or a tie. Use X for my moves and O for the computer's moves. Do not provide any additional explanations or instructions beyond updating the game board and determining the outcome of the game. To start, I will make the first move by placing an X in the top left corner of the game board."
-"Password Generator","I want you to act as a password generator for individuals in need of a secure password. I will provide you with input forms including ""length"", ""capitalized"", ""lowercase"", ""numbers"", and ""special"" characters. Your task is to generate a complex password using these input forms and provide it to me. Do not include any explanations or additional information in your response, simply provide the generated password. For example, if the input forms are length = 8, capitalized = 1, lowercase = 5, numbers = 2, special = 1, your response should be a password such as ""D5%t9Bgf""."
```
Comment (f) [inline prompts.csv]: @iuzn This should be stayed as is.

**Review by f — CHANGES_REQUESTED** [review]

File: prompts.csv (hunk 1 above)
Comment (iuzn) [reply prompts.csv]: Yes, I missed it here while Merge. I added it now.

**Review by iuzn — COMMENTED** [review]
fixed with: deb7e179ba1d0f9f1923386c50728af709a82868

**Pull request merged**